.github/workflows/build-test.yaml

```diff
@@ -641,10 +641,10 @@ jobs:
       fail-fast: false
       matrix:
         cluster: [
-          {distribution: k3s, version: v1.24},
           {distribution: k3s, version: v1.25},
           {distribution: k3s, version: v1.26},
-          {distribution: eks, version: v1.27}
+          {distribution: eks, version: v1.27},
+          {distribution: k3s-local, version: v1.28} # TODO: use CMX for this when 1.28 support is added
```

Comment:
i think the version for `k3s-local` needs to be a valid k3s version - see `version-history-pagination`.  looking at https://github.com/k3s-io/k3s, i'm not sure they've added support for 1.28 yet?